content/codes/components/enigma.pug

```diff
@@ -1,9 +1,59 @@
 template#enigma
-  .enigma
-    .lightboard
-      for key in 'qwertyuiopasdfghjklzxcvbnm'.split('')
-        .key= key
+    .enigma#enigmaview
+      .lightboard#lightboard
+        for key in 'QWERTZUIO'.split('')
+          .key= key
+        for key in 'ASDFGHJK'.split('')
+          .key= key
+        for key in 'YXCVBNML'.split('')
+          .key= key
 
-    .keyboard
-      for key in 'qwertyuiopasdfghjklzxcvbnm'.split('')
-        .key= key
+      .keyboard#keyboard
+        for key in 'QWERTZUIO'.split('')
+          .key= key
+        for key in 'ASDFGHJK'.split('')
+          .key= key
+        for key in 'YXCVBNML'.split('')
+          .key= key
+  
+      .plugboard#plugboard
+          <svg id="target">			
+            <path class="plugcable" id="path0" />
+            <path class="plugcable" id="path1" />
+            <path class="plugcable" id="path2" /> 
+            <path class="plugcable" id="path3" />
+            <path class="plugcable" id="path4" />
+            <path class="plugcable" id="path5" />
+            <path class="plugcable" id="path6" />
+            <path class="plugcable" id="path7" />
+            <path class="plugcable" id="path8" />
+            <path class="plugcable" id="path9" />
```

Comment:
Let's use loops here (and the same below), e.g.
```
for i in [0, 1, 2, 3, 4, 5, 6, 7, 8, 9]
  path.plugcable(class="path" + i)
```

However, I also don't think we should give unique classes to each of the elements – we could just do `$('#target').children`.